Lib/test/test_pyexpat.py

```diff
@@ -506,6 +506,7 @@ def EndElementHandler(self, name):
     def check_pos(self, event):
         pos = (event,
                self.parser.CurrentByteIndex,
+               self.parser.CurrentByteCount,
```

Comment:
Should we have a dedicated test as well for this one? The libexpact [docs](https://libexpat.github.io/doc/api/latest/#XML_GetCurrentByteCount) say:

> Returns 0 if the event is inside a reference to an internal entity and for the end-tag event for empty element tags (the later can be used to distinguish empty-element tags from empty elements using separate start and end tags

Reply:
The test case already includes an example of an _empty-element_  as defined in the XML spec[1]. What is missing is an example of an empty element in the long form (with a separate open and close tag), but that seems redundant.

I'll gladly add it if you'd like to explicitly include it.

[1]: https://www.w3.org/TR/xml/#dt-empty